src/user/system/LoadBalancerManager.ts

```diff
@@ -532,7 +532,7 @@ class LoadBalancerManager {
                         serviceName: CaptainConstants.captainServiceName,
                         domain: captainDomain,
                         serviceExposedPort:
-                            CaptainConstants.configs.adminPortNumber3000,
+                            CaptainConstants.captainServiceExposedPort,
```

Comment:
Let's just hardcode this to 3000. Or, change [the server port](https://github.com/caprover/caprover/blob/master/src/server.ts#L33) to use this constant as well - they need to stay in sync.

Reply:
By the way, this is no longer the "Exposed port", we should rename it to `captainServiceContainerPort`

Reply:
Done it. Minimal changes and a single source of truth for 3000. Except docker files and so on. But this is not supposed to ever change.